exercises/Scientific-1-R.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: exercise
+title: Scientific 1
+language: R
+---
+
+There were a relatively large number of extinctions of mammalian species
+roughly 10,000 years ago. To help understand why these extinctions
+happened scientists are interested in understanding whether there were
+differences in the body size of those species that went extinct and
+those that did not. Since we're starting to get pretty good at this
+whole programming thing let's stop messing around with made up datasets
+and do some serious analysis.
+
+Download the
+[largest dataset on mammalian body size in the world](http://www.esapubs.org/archive/ecol/E084/094/#data).
+Fortunately this dataset has data on the mass of recently extinct mammals as
+well as extant mammals (i.e., those that are still alive today). Take a look at
+the [metadata](http://www.esapubs.org/archive/ecol/E084/094/metadata.htm) to
+understand the structure of the data. One key thing to remember is that species
+can occur on more than one continent, and if they do then they will occur more
+than once in this dataset. Also let's ignore species that went extinct in the
+very recent past (designated by the word 'historical' in the 'status' column).
+
+Import the data into R. If you've looked at a lot of data you'll realize
+that this dataset is tab delimited. Use the argument `sep = ‘’` in `read.csv()` to properly format the data. There is no header row.
```

Comment:
>> use sep = "\t".
Done.

Also, updated language of items 1 and 5 to match solution.

On Tue, May 12, 2015 at 10:15 AM, Ethan White <notifications@github.com>
wrote:

> In exercises/Scientific-1-R.md
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/50#discussion_r30144523>
> :
>
> > +those that did not. Since we're starting to get pretty good at this
> > +whole programming thing let's stop messing around with made up datasets
> > +and do some serious analysis.
> > +
> > +Download the
> > +[largest dataset on mammalian body size in the world](http://www.esapubs.org/archive/ecol/E084/094/#data).
> > +Fortunately this dataset has data on the mass of recently extinct mammals as
> > +well as extant mammals (i.e., those that are still alive today). Take a look at
> > +the [metadata](http://www.esapubs.org/archive/ecol/E084/094/metadata.htm) to
> > +understand the structure of the data. One key thing to remember is that species
> > +can occur on more than one continent, and if they do then they will occur more
> > +than once in this dataset. Also let's ignore species that went extinct in the
> > +very recent past (designated by the word 'historical' in the 'status' column).
> > +
> > +Import the data into R. If you've looked at a lot of data you'll realize
> > +that this dataset is tab delimited. Use the argument `sep = ‘’` in `read.csv()` to properly format the data. There is no header row.
>
> sep = ‘’ won't actually work because those are special quote characters
> (at least it fails for me). Regardless I think it's better to be explicit
> here and use sep = '\t'.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethanwhite/datacarp-semester-biology/pull/50/files#r30144523>
> .
>